### PR TITLE
refactor activemq-5.18.3 role to user alpine-jre

### PR DESCRIPTION
**Pull request by karolisdzekunskas**

### CODE REVIEW

File: ActiveMQ-classic/5.18.3-jdbc/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.env
```
Comment (karolisdzekunskas) [inline ActiveMQ-classic/5.18.3-jdbc/.gitignore:1]: remove this

File: ActiveMQ-classic/5.18.3-jdbc/dockerImage/Dockerfile [hunk 2]
```diff
@@ -57,6 +54,6 @@ RUN chmod 777 /docker-entrypoint.sh
 
 WORKDIR ${ACTIVEMQ_WORKDIR}/apache-activemq-${AMQ_VERSION}/
 
-EXPOSE 8161 61616
+EXPOSE 8161 12345 61616
```
Comment (karolisdzekunskas) [inline ActiveMQ-classic/5.18.3-jdbc/dockerImage/Dockerfile:57]: check if this relevant

File: ActiveMQ-classic/5.18.3-jdbc/dockerImage/assets/config.yaml [hunk 3]
```diff
@@ -1,47 +1,2 @@
...
-  - "org.apache.activemq:destinationType=Queue,*"
-  - "org.apache.activemq:destinationType=Topic,*"
-  - "org.apache.activemq:type=Broker,brokerName=*"
-  - "org.apache.activemq:type=Topic,brokerName=*"
-  - "org.apache.activemq:type=Broker,brokerName=*,service=*"
-
-# @ToDo take host from ENV
-hostPort: localhost:1099
-username: monitorRole
-password: MONITOR_ROLE_PASS
```
Comment (karolisdzekunskas) [inline ActiveMQ-classic/5.18.3-jdbc/dockerImage/assets/config.yaml:16]: relevant

File: ActiveMQ-classic/5.18.3-jdbc/dockerImage/assets/docker-entrypoint.sh [hunk 4]
```diff
@@ -1,43 +1,61 @@
 #!/bin/sh
 set -x
```
Comment (karolisdzekunskas) [inline ActiveMQ-classic/5.18.3-jdbc/dockerImage/assets/docker-entrypoint.sh:2]: think how to mask secrets

File: ActiveMQ-classic/5.18.3-jdbc/dockerImage/assets/docker-entrypoint.sh [hunk 5]
```diff
@@ -1,43 +1,61 @@
...
-sed -i "s/MONITOR_ROLE_PASS/${MONITOR_ROLE_PASS}/" $ACTIVEMQ_WORKDIR/apache-activemq-5.18.3/conf/jmx.password
-sed -i "s/CONTROL_ROLE_PASS/${CONTROL_ROLE_PASS}/" $ACTIVEMQ_WORKDIR/apache-activemq-5.18.3/conf/jmx.password
+sed -i "s/USE_JMX/${USE_JMX}/" /opt/apache-activemq-${AMQ_VERSION}/conf/activemq.xml
+sed -i "s/127.0.0.1/0.0.0.0/g" $ACTIVEMQ_WORKDIR/apache-activemq-${AMQ_VERSION}/conf/jetty.xml
+sed -i "s/MONITOR_ROLE_PASS/${MONITOR_ROLE_PASS}/" $ACTIVEMQ_WORKDIR/apache-activemq-${AMQ_VERSION}/conf/jmx.password
+sed -i "s/CONTROL_ROLE_PASS/${CONTROL_ROLE_PASS}/" $ACTIVEMQ_WORKDIR/apache-activemq-${AMQ_VERSION}/conf/jmx.password
 sed -i "s/MONITOR_ROLE_PASS/${MONITOR_ROLE_PASS}/" $CONTAINER_METRICS/config.yaml
-if [ "$OPENWIRE_ENABLED" = true ] ; then
-sed -i "/\t<transportConnectors\>/a\\ \t\t<transportConnector name=\"openwire\" uri=\"tcp:\/\/0.0.0.0:$OPENWIRE_PORT?maximumConnections=1000\&amp;wireFormat.maxFrameSize=104857600\"\/\>" /opt/apache-activemq-5.18.3/conf/activemq.xml
+# OPENWIRE is enabled by default so FALSE statement deletes line
```
Comment (karolisdzekunskas) [inline ActiveMQ-classic/5.18.3-jdbc/dockerImage/assets/docker-entrypoint.sh:25]: activemq.xml exist, check this

File: ActiveMQ-classic/5.18.3-jdbc/dockerImage/assets/docker-entrypoint.sh (hunk 4 above)
Comment (karolisdzekunskas) [reply ActiveMQ-classic/5.18.3-jdbc/dockerImage/assets/docker-entrypoint.sh:2]: possible only with `command > /dev/null 2>&1`

**Review by eimantasLinartas — APPROVED** [review]

**Pull request merged**